packages/frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,37 @@
+import { TruefiLogoIcon } from 'src/components/Icons'
+import { Colors, hexOpacity } from 'src/styles/colors'
+import styled from 'styled-components'
+
+export const Footer = () => {
+  return (
+    <FooterContainer>
+      <FooterRow>
+        <p>Built with 💙 by</p>
+        <TruefiLogoIcon />
+      </FooterRow>
+      <FooterCopyright>© 2022. All rights reserved.</FooterCopyright>
```

Comment:
Shouldn't it say something like "© 2022 [copyright owner]"?  Or do we not know the owner yet?